pkg/archive/archive_test.go

```diff
@@ -700,6 +701,56 @@ func tarUntar(t *testing.T, origin string, options *TarOptions) ([]Change, error
 	return ChangesDirs(origin, tmp)
 }
 
+func DecompressWithZstd(in []byte) []byte {
+	input := bytes.NewBuffer(in)
+	output := &bytes.Buffer{}
+	d, err := zstd.NewReader(input)
+	if err != nil {
+		return output.Bytes()
+	}
+	defer d.Close()
+	io.Copy(output, d)
+	return output.Bytes()
+}
+
+func CompressWithZstd(in []byte) []byte {
+	input := bytes.NewBuffer(in)
+	output := &bytes.Buffer{}
+	enc, _ := zstd.NewWriter(output)
+	io.Copy(enc, input)
+	enc.Close()
+	return output.Bytes()
+}
+
+func TestDetectCompression(t *testing.T) {
+	// test zstd compression without skippable frames.
+	compressedData := CompressWithZstd([]byte("hello world"))
```

Comment:
Perhaps we should just use a fixed value for this as well (instead of generating a zstd compressed string on the fly)

Reply:
Sounds good.